### PR TITLE
Switch to the common CoC

### PR DESCRIPTION
**Pull request by jsynowiec**
Related to https://github.com/nodejs/community-committee/issues/34, updates the CoC so it points to one place.

### CODE REVIEW

**Review by vdeturckheim — APPROVED** [review]

**Review by bnb — APPROVED** [review]

**Comment by bnb** [comment]
Per the "[Consensus Seeking](https://github.com/nodejs/evangelism/blob/master/GOVERNANCE.md#consensus-seeking-process)" section in the Governance doc, we do seem to have consensus so far - I will leave this open for a few more days and request that members of @nodejs/evangelism review and approve, request changes, or reject if y'all have time. If there are no objections, we can merge this 🙏

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,37 +28,6 @@ By making a contribution to this project, I certify that:
...
-  the harassment if possible.  Whether you're a regular contributor or
-  a newcomer, we care about making this community a safe place for you
-  and we've got your back.
-* Likewise any spamming, trolling, flaming, baiting or other
-  attention-stealing behaviour is not welcome.
-* Avoid the use of personal pronouns in code comments or
-  documentation. There is no need to address persons when explaining
-  code (e.g. "When the developer")
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node Foundation requires of its contributors.
```
Comment (thefourtheye) [inline CONTRIBUTING.md]: Nit: Node.js Foundation.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,37 +28,6 @@ By making a contribution to this project, I certify that:
...
+[Please read it before participating.](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md)
```
Comment (thefourtheye) [inline CONTRIBUTING.md]: Why not the CoC doc in the TSC repo? This target asks users to go there anyway.

File: CONTRIBUTING.md (hunk 1 above)
Comment (thefourtheye) [reply CONTRIBUTING.md]: I am not sure if the usage is correct, but I always thought "requires off of" is the correct form. Or how about "requires from"?

**Review by thefourtheye — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (bnb) [reply CONTRIBUTING.md]: Looking at the current [CommComm version](https://github.com/nodejs/community-committee/blob/master/CODE_OF_CONDUCT.md), I'm inclined to agree. Should just be a simple change of URL 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (jsynowiec) [reply CONTRIBUTING.md]: It follows the CommComm's [CONTRIBUTING.md#code-of-conduct](https://github.com/nodejs/community-committee/blob/master/CONTRIBUTING.md#code-of-conduct). I'm going to update it and also send a PR to CommComm's repository.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by bnb** [comment]
Going to merge. Thanks @jsynowiec! 🙏

**Pull request merged**